client/src/components/CommentApp/components/Comment/index.tsx

```diff
@@ -629,6 +629,11 @@ export default class CommentComponent extends React.Component<CommentProps> {
                 }),
               );
             },
+            // Allow delay for side panel to open and comment card to fade in with animations
+            checkCanFocusTrap: () =>
+              new Promise((resolve) => {
+                setTimeout(resolve, 250);
+              }),
             initialFocus: '[data-focus-target="true"]',
             delayFocus: false,
```

Comment:
Let's remove this `delayFocus`, as noted, it's not in the API for focus-trap any more and the default is probably what we want.

```suggestion
```

Alternatively, be explicit that we do want the default.

```suggestion
            delayInitialFocus: true,
```


> delayInitialFocus {boolean}: Default: true. Delays the autofocus to the next execution frame when the focus trap is activated. This prevents elements within the focusable element from capturing the event that triggered the focus trap activation.